dietpi/dietpi-config

```diff
@@ -643,7 +643,7 @@ A long (or insufficiently manufactured) cable may required a higher boost settin
    Source: https://www.raspberrypi.org/forums/viewtopic.php?p=105008#p105008\n
 Re-enabling HDMI requires a reboot. If you need emergency HDMI output, edit the following files on first partition of the SDcard from external system:
  - In config.txt, set "hdmi_ignore_hotplug=0" and comment/remove all (max_)framebuffer_(width/height) lines.
```

Comment:
There is another outdated info, as we disable the framebuffer nowadays instead of minimising its resolution:
```suggestion
 - In config.txt, set "hdmi_ignore_hotplug=0" and comment/remove the "max_framebuffers=0" line.
```